src/Apps.ts

```diff
@@ -1,20 +1,22 @@
-import * as App from "./App";
+import AppContext from "./Apps/AppContext";
 import * as Document from "./Document";
 import Context from "./Context";
-import * as HTTP from "./HTTP";
+import * as Response from "./HTTP/Response";
 import * as Pointer from "./Pointer";
 import * as RDF from "./RDF";
 import * as Utils from "./Utils";
-import * as CS from "./NS/CS";
+import * as App from "./Apps/App";
+import * as PersistedApp from "./Apps/PersistedApp";
+import * as Errors from "./Errors";
 
-class Apps {
+export class Class {
 	private context:Context;
 
 	constructor( context:Context ) {
 		this.context = context;
 	}
 
-	get( uri:string ):Promise<App.Context> {
+	getAppContext( uri:string ):Promise<AppContext> {
```

Comment:
@MiguelAraCo should I also change the others methods?
Leaving:

* `getAppContext` => `getContext`
* `getAllAppContext` => `getAllContext`
* `createApp` => `create`